MathLib/LinAlg/Lis/LisOption.h

```diff
@@ -40,12 +41,13 @@ namespace MathLib
  */
 struct LisOption
 {
-	LisOption(BaseLib::ConfigTree const* const options)
+	LisOption(BaseLib::ConfigTreeNew const* const options)
 	{
 		if (options)
 		{
-			_option_string += " " + options->get<std::string>("lis", "");
-			boost::algorithm::trim_right(_option_string);
+			ignoreOtherLinearSolvers(*options, "lis");
+			if (auto s = options->getConfParamOptional<std::string>("lis"))
+				_option_string += " " + *s;
```

Comment:
If `*s` is an empty string, the `_option_string` will grow by a whitespace. Therefore it was trimmed. If this is handled differently now, remove boost include from the file.